.ruby-gemset

```diff
@@ -0,0 +1 @@
+prawn-fillform
```

Comment:
This and .ruby-version are here to make this repo play better with rvm. This tells RVM to make a new gemset for this repo.